frontend/src/Pages/FeedPage.tsx

```diff
@@ -5,65 +5,64 @@ import PostComponent from '../Components/PostComponent';
 import Paginator from '../Components/Paginator';
 import {Link, useMatch, useSearchParams} from 'react-router-dom';
 import {FeedType, useFeed} from '../API/use/useFeed';
+import {APIError} from '../API/APIBase';
 
 export default function FeedPage() {
```

Comment:
`FeedPage` must be made `observer`, as otherwise it doesn't react to the `site` change (noticeable, if you click on subsites in the left panel).

Reply:
with https://github.com/spaceshelter/orbitar/commit/11a044609380218ec23d3806a578c76338bcbc6d `observer` is not strictly required, as `useSearchParams` and `useMatch` hooks trigger the render of `FeedPage` on location change anyway, but it may be needed in the future.

Reply:
Fixed in ad82828192517f0701ef3dc9b7a62e4a0cc0f3c6